diff --git a/uefi-sct/SctPkg/CommonGenFramework.sh b/uefi-sct/SctPkg/CommonGenFramework.sh
index c42e8904..28c3a438 100755
--- a/uefi-sct/SctPkg/CommonGenFramework.sh
+++ b/uefi-sct/SctPkg/CommonGenFramework.sh
@@ -411,6 +411,15 @@ then
     cp $ProcessorType/ImageServicesBBTest.efi                  $Framework/Test/ > NUL
     cp $ProcessorType/MiscBootServicesBBTest.efi               $Framework/Test/ > NUL
 
+    # BBSR
+    cp $ProcessorType/TCGMemoryOverwriteRequestBBTest.efi      $Framework/Test/ > NUL
+    cp $ProcessorType/TCG2ProtocolBBTest.efi                   $Framework/Test/ > NUL
+  if [ $SECUREBOOT_ENABLE -eq 1 ]; then
+    cp $ProcessorType/SecureBootBBTest.efi                     $Framework/Test/ > NUL
+  fi
+
     cp $ProcessorType/VariableServicesBBTest.efi               $Framework/Test/ > NUL
     cp $ProcessorType/TimeServicesBBTest.efi                   $Framework/Test/ > NUL
     cp $ProcessorType/MiscRuntimeServicesBBTest.efi            $Framework/Test/ > NUL
@@ -569,4 +578,7 @@ then
     CopyDependency PciRootBridgeIo
     CopyDependency PxeBaseCode
     CopyDependency ConfigKeywordHandler
+  if [ $SECUREBOOT_ENABLE -eq 1 ]; then
+    CopyDependency SecureBoot
+  fi
 fi
